WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -253,6 +253,47 @@ public object[] GetHistory()
 			}).ToArray();
 	}
 
+	[JsonRpcMethod("excludefromcoinjoin")]
+	public IEnumerable<OutPoint> ExcludeFromCoinjoin(bool exclude, string? password = null, OutPoint[]? outPoints = null, string[]? labels = null)
```

Comment:
Please design the API some how, this is not good and I am pretty sure it doesn't work. Have you tested it?

It seems to me this method tries to do too much, in fact it tries to do two things:
* exclude by outpoint. OK
* exclude by labels. Not OK. RPC is an interface to build things. 

Anyway, even if exclude by labels is desired, the interface would be really complex so, it would be better to have another method.

Reply:
It was working; just I forgot to call the update. Anyway, I made this draft PR exactly to have this kind of feedback about the signature/implementation, so thanks.